.jenkins/pytorch/win-test-helpers/setup_pytorch_env.bat

```diff
@@ -40,7 +40,7 @@ popd
 pip install "ninja==1.10.0.post1" future "hypothesis==4.53.2" "expecttest==0.1.3" "librosa>=0.6.2" psutil pillow unittest-xml-reporting pytest
 
 :: Only the CPU tests run coverage, which I know is not super clear: https://github.com/pytorch/pytorch/issues/56264
-if "%BUILD_ENVIRONMENT%" == "pytorch-win-vs2019-cpu-py3" (
+if "%BUILD_ENVIRONMENT%" == "win-vs2019-cpu-py3" (
```

Comment:
```suggestion
if "%BUILD_ENVIRONMENT%" == "*win-vs2019-cpu-py3*" (
```
I noticed you put stars around the other edits--should this one have them too?

Reply:
good catch. but I dunno if this will work on bat script. and we only run windows jobs on GHA now so it is fine.

Reply:
We do still have 1 job on CircleCI so it might be good to check,

Judging from https://superuser.com/questions/1452381/regular-expression-in-batch/1452399 it looks like it's possible with:

```suggestion
echo "%BUILD_ENVIRONMENT%" | findstr /R ".*win-vs2019-cpu-py3.*" (
```

Docs: https://ss64.com/nt/findstr.html